src/db-service.ts

```diff
@@ -141,34 +153,48 @@ export class ItemValidationService {
         FROM item_validation_review AS ivr
         INNER JOIN item_validation AS iv
         ON ivr.item_validation_id = iv.id
-        INNER JOIN item_validation_process AS ivp
-        ON iv.item_validation_process_id = ivp.id
         ORDER BY ivr.created_at ASC
       `,
       )
       .then(({ rows }) => rows.slice());
   }
 
   /**
-   * Create an entry for the automatic validation process in DB
+   * Create an entry for the validation attempt in item-validation
+   * @param itemId id of the item being validated
+   */
+  async createItemValidation(itemId: string, transactionHandler: TrxHandler): Promise<string> {
+    return transactionHandler
+      .query<{ id: string }>(
+        sql`
+          INSERT INTO item_validation (item_id)
+          VALUES (
+            ${itemId}
+          )
+          RETURNING *
+        `,
+      )
+      .then(({ rows }) => rows[0]?.id);
+  }
+
+  /**
+   * Create an entry for the automatic validation process in item-validation-group
    * @param itemId id of the item being validated
-   * @param processId id of the validation process
+   * @param item-validation-id
```

Comment:
Add type as in js doc. It would be nice to provide explanations for all parameters if you add the doc, and refer to the same variable name.